src/pages/summary/Summary.tsx

```diff
@@ -11,9 +11,23 @@ import "./Summary.scss";
 
 const contentElementId: string = "main-content";
 
-class Summary extends React.Component<{}, {}> {
+class Summary extends React.Component<{}, { clusterName: string }> {
+    public constructor(props: {}) {
+        super(props);
+        this.state = { clusterName: "" }
+        fetch("/clusterName")
+            .then((res) => {
+                res.json().then((val) => {
+                    this.setState({
+                        clusterName: val.clusterName
+                    });
+                });
+            });
+    }
+
     public render(): JSX.Element {
-        return <K8sSummary.KubeSummary kubeService={this._pageDataService} title="Kubernetes summary UI Web App" />;
+        const props = { title: "Kubernetes summary UI Web App", kubeService: this._pageDataService, clusterName: this.state.clusterName } as K8sSummary.IKubeSummaryProps;
+        return <K8sSummary.KubeSummary  {...props} />;
```

Comment:
KubeSummaryProps is not having clusterName. so today even if pass, if k8s UI does not honor then we can not show, is this right?

Reply:
Yes k8s UI should be updated